components/tikv_util/src/future_pool/builder.rs

```diff
@@ -58,7 +58,7 @@ impl Builder {
     where
         F: Fn() + Send + Sync + 'static,
     {
-        self.inner_builder.after_start(f);
+        self.inner_builder.after_start(add_thread_name!(f));
```

Comment:
Do we need to tweak `before_stop`? Since threads may vary dynamically in future.

Reply:
done